app/models/post.rb

```diff
@@ -0,0 +1,156 @@
+
+            When the Travellers came into his presence he asked them what they thought of him as a King.
+
+            The lying Traveller said, "Sire, every one must see that you are a most noble and mighty monarch."
+
+            "And what do you think of my subjects?" continued the King.
+
+            "They," said the Traveller, "are in every way worthy of their royal master."
+
+            The Ape was so delighted with his answer that he gave him a very handsome present.
+
+            The other Traveller thought that if his companion was rewarded so splendidly for telling a lie, he himself would certainly receive a still greater reward for telling the truth; so, when the Ape turned to him and said, "And what, sir, is your opinion?" he replied, "I think you are a very fine Ape, and all your subjects are fine Apes too."
+
+            The King of the Apes was so enraged at his reply that he ordered him to be taken away and clawed to death.
+          ),
+          created_at:   rand(6).months.ago,
+          image: 'footer4'
+        ),
+        Post.new(
+          id: 5,
+          title:  "Dr. Strangelove (or, how I learned to stop worrying and love the bomb)",
+          author: "Stanley Kubrick",
+          body: %q(
+            Two men were travelling together, one of whom never spoke the truth, whereas the other never told a lie: and they came in the course of their travels to the land of Apes.
+
+            The King of the Apes, hearing of their arrival, ordered them to be brought before him; and by way of impressing them with his magnificence, he received them sitting on a throne, while the Apes, his subjects, were ranged in long rows on either side of him.
+
+            When the Travellers came into his presence he asked them what they thought of him as a King.
+
+            The lying Traveller said, "Sire, every one must see that you are a most noble and mighty monarch."
+
+            "And what do you think of my subjects?" continued the King.
+
+            "They," said the Traveller, "are in every way worthy of their royal master."
+
+            The Ape was so delighted with his answer that he gave him a very handsome present.
+
+            The other Traveller thought that if his companion was rewarded so splendidly for telling a lie, he himself would certainly receive a still greater reward for telling the truth; so, when the Ape turned to him and said, "And what, sir, is your opinion?" he replied, "I think you are a very fine Ape, and all your subjects are fine Apes too."
+
+            The King of the Apes was so enraged at his reply that he ordered him to be taken away and clawed to death.
+          ),
+          created_at:   rand(6).months.ago,
+          image: 'footer5'
+        )
+      ]
+    end
+    @posts
+  end
+
+  def self.find(id)
+    all.detect { |post| post.id == id }
+  end
+
+  def self.sorted
+    all.sort_by { |post| Date.today - post.created_at.to_i }
```

Comment:
I suspect you might have been able to just `sort_by { |post| post.created_at }`. Might have needed a `.reverse` at the end, though.